docs/usage.md

```diff
@@ -193,18 +197,18 @@ A location does not have a prescribed shape. In your UI it can represent a singl
 
 The location property will be set on the [member](#members).
 
-Because locations are part of members, a `memberUpdate` event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
+Because locations are part of members, an `update` member event will be emitted when a member updates their location. When a member leaves, their location is set to `null`.
 
 ```ts
-space.subscribe('membersUpdate', (members) => {
-  console.log(members);
+space.members.subscribe('update', (memberProfileUpdate) => {
```

Comment:
It's not just a profile update. This will get triggered for enters, leaves, removes also.